### ISSUE TITLE
Maybe a wrong version

### ISSUE THREAD

**Issue by ZeweiLi999** [jinyeying/DeS3_Deshadow#3]
Hi Jin Yeying,

Thank you for your wonderful work. However, I can't find the attention map loss and ViT loss in the code. Perhaps you uploaded the wrong version to GitHub.

Looking forward to receiving the correct version of the code.

**Comment by jinyeying** [comment]
> Hi Jin Yeying,
> 
> Thank you for your wonderful work. However, I can't find the attention map loss and ViT loss in the code. Perhaps you uploaded the wrong version to GitHub.
> 
> Looking forward to receiving the correct version of the code.

yes, you are right. I have not cleaned the attention map loss and ViT loss code yet.

**Comment by atsatvik** [comment]
Hi.

I was wondering if you have updated the code yet?

**Comment by jinyeying** [comment]
> Hi.
> 
> I was wondering if you have updated the code yet?

the aistd version is a basic version without losses version. you can simply use it for training and inference.

**Comment by atsatvik** [comment]
I am trying to recreate the exact results from the paper, will I be able to do that using the current codebase?

**Comment by atsatvik** [comment]
Hey, I'm trying to build upon your work and I would greatly appreciate if you can provide me with the necessary additions (like the ViT similarity and attention loss) that I need to make to recreate the results from the paper, thank you!

**Comment by atsatvik** [comment]
I have a question: 
Since we are using a noisy shadow image -> feed it into the diffusion network -> get noise removed + shadow removed image

This image should ideally not have any shadows and if it's perfect then the classification label will be 1 (no shadows) otherwise it will be zero (shadow present), is my understanding correct? 

And if this is the case then how do we decide whether the image has no shadow whatsoever, do we compare the diffusion model output image with shadow image and see how close it is and then set the classification label accordingly. Is that what is happening here?
